middleware.md

```diff
@@ -250,7 +250,9 @@ Middleware parameters may be specified when defining the route by separating the
 <a name="terminable-middleware"></a>
 ## Terminable Middleware
 
-Sometimes a middleware may need to do some work after the HTTP response has been prepared. For example, the "session" middleware included with Laravel writes the session data to storage after the response has been fully prepared. If you define a `terminate` method on your middleware, it will automatically be called after the response is ready to be sent to the browser.
+Sometimes a middleware may need to do some work after the HTTP response has been sent to the browser. For example, the "session" middleware included with Laravel writes the session data to storage after the response has been sent to the browser. If you define a `terminate` method on your middleware, it will automatically be called after the response is sent to the browser.
+
+> **Note:** If your web server is not configured to use FastCGI, the `terminate` method on your middleware will be called after the response is prepared to be sent to the browser.
```

Comment:
```suggestion
> **Note:** If your web server is not configured to use FastCGI, the request will run the `terminate` method on your middleware before the HTTP response is sent to the browser. This may result in an increased HTTP response time as it will wait for the code to complete its processing in the `terminate` method.
```